core/src/test/java/org/jdbi/v3/core/TestJsr310.java

```diff
@@ -45,49 +45,49 @@ public void setUp() {
     public void instant() {
         Instant i = Instant.now();
         h.insert("insert into stuff(ts) values (?)", i);
-        assertEquals(i, h.createQuery("select ts from stuff").mapTo(Instant.class).findOnly());
+        assertThat(h.createQuery("select ts from stuff").mapTo(Instant.class).findOnly()).isEqualTo(i);
     }
 
     @Test
     public void localDate() {
         LocalDate d = LocalDate.now();
         h.insert("insert into stuff(d) values (?)", d);
-        assertEquals(d, h.createQuery("select d from stuff").mapTo(LocalDate.class).findOnly());
+        assertThat(h.createQuery("select d from stuff").mapTo(LocalDate.class).findOnly()).isEqualTo(d);
     }
 
     @Test
     public void localDateTime() {
         LocalDateTime d = LocalDateTime.now();
         h.insert("insert into stuff(ts) values (?)", d);
-        assertEquals(d, h.createQuery("select ts from stuff").mapTo(LocalDateTime.class).findOnly());
+        assertThat(h.createQuery("select ts from stuff").mapTo(LocalDateTime.class).findOnly()).isEqualTo(d);
     }
 
     @Test
     public void offsetDateTime() {
         OffsetDateTime dt = OffsetDateTime.now();
         h.insert("insert into stuff(ts) values (?)", dt);
-        assertEquals(dt, h.createQuery("select ts from stuff").mapTo(OffsetDateTime.class).findOnly());
+        assertThat(h.createQuery("select ts from stuff").mapTo(OffsetDateTime.class).findOnly()).isEqualTo(dt);
     }
 
     @Test
     public void offsetDateTimeLosesOffset() {
         OffsetDateTime dt = OffsetDateTime.now().withOffsetSameInstant(ZoneOffset.ofHours(-7));
         h.insert("insert into stuff(ts) values (?)", dt);
-        assertTrue(dt.isEqual(h.createQuery("select ts from stuff").mapTo(OffsetDateTime.class).findOnly()));
+        assertThat(h.createQuery("select ts from stuff").mapTo(OffsetDateTime.class).findOnly().isEqual(dt)).isTrue();
```

Comment:
I think this would now work when the database in other timezone. I get:
```
java.lang.AssertionError: 
Expecting:
  <2016-10-05T17:32:34.260Z>
to have same time fields except timezone as:
  <2016-10-05T10:32:34.260-07:00>
but had not.
```